Bump RC to version 0.1.6

bleehu opened the pull request:
https://areyoudownwithotpyesyouknowme.tumblr.com/post/48641430196

bleehu committed:
added some rules about attack dogs

bleehu committed:
added tricks

bleehu committed:
Merge pull request #481 from bleehu/MH/Hunter-Dog/Add-tricks-doc

added tricks

bleehu committed:
Revert "added tricks"

bleehu committed:
Merge pull request #485 from bleehu/revert-481-MH/Hunter-Dog/Add-tricks-doc

Revert "added tricks"

bleehu committed:
Merge pull request #482 from bleehu/MH/add-dog-rules

added some rules about attack dogs

Turtlelord26 committed:
Melee weapons no longer have explosive damage

Sledgehammer and Power Fist now deal regular ballistic damage ... or would if any melee weapon specified its damage type.

Scope crept on instances of skill being used in damage calculations of thrown weapons. They are gone now, per #508

Turtlelord26 committed:
Clarify Explosive damage

Is now not also concussive damage. 'Concussive' as a damage is now effectively deleted from Compound X. Individual effects that want to do unusual things re armor but are not explosive must be given prose rule exception text in their notes sections.

bleehu committed:
Added Hunter's Dog document,
As well as basic rules for dog and reference in classes doc. Added warning about projection
onto the dog. Attempting to keep dog mechanics very simple. Though I don't think I've
achieved my goal of making the hunter class feel like predator.

Changes to be committed:
	modified:   02_Classes.txt
	new file:   Class-Specific Documentation/Hunters_Dog.txt

bleehu committed:
Merge branch 'dev' into MH/Hunter-Dog/Superbranch

bleehu committed:
added dog food item

bleehu committed:
added dog toy item

bleehu committed:
added dog collar/leash

bleehu committed:
changed description of hunter class; no longer an emphasis on nanite usage

bleehu committed:
Removed old hunter nanite class feats. Added Tracker, Spotter and Pack Hunter feats
	modified:   CharacterCreation/02_Classes.txt

bleehu committed:
Merge pull request #583 from bleehu/MH/Hunter-Dog/DogStats

Mh/hunter dog/dog stats

bleehu committed:
Merge pull request #584 from bleehu/MH/Hunter-Dog/Dogfood

added dog food item

bleehu committed:
Merge pull request #587 from bleehu/MH/Hunter-Dog/description

changed description of hunter class

bleehu committed:
Changed collar to harness

bleehu committed:
removed 'crit' from hunter class feats

bleehu committed:
Merge pull request #588 from bleehu/MH/Hunter-Dog/ClassFeats

Mh/hunter dog/class feats

Turtlelord26 committed:
One sentence structure note

bleehu committed:
removed line about getting attention; trained attack dogs don't need
toys for getting attention.

bleehu committed:
Merge pull request #586 from bleehu/MH/Hunter-Dog/Collar

added dog collar/leash

bleehu committed:
Merge pull request #585 from bleehu/MH/Hunter-Dog/chewtoy

added dog toy item

bleehu committed:
removed sniper rifle from hunter class

bleehu committed:
added new class feat to hunters: Matador

bleehu committed:
changed name of hunter class feat from Matatdor to Bushwhacker

bleehu committed:
changed had to wielding in bushwacker feat for hunter

bleehu committed:
Merge pull request #617 from bleehu/MH/Hunter-Dog/Matador

Mh/hunter dog/matador

Masterdiekillsu committed:
Updated "miss chance", to "Accuracy"

Changed all instances of the usage of the words "miss chance", to "accuracy" in order to reflect the current Combat 2.0 changes.

Masterdiekillsu committed:
Merge pull request #627 from bleehu/NY-Feat-Wording

Updated "miss chance", to "Accuracy"

Turtlelord26 committed:
Typography

oops.

Turtlelord26 committed:
Typo. Them -> Then in "Firing into Melee Combat"

Turtlelord26 committed:
Added Welding Torch, added #s to descriptions

Added Eliazar Welder, identical to the Plasma Torch but with Fire damage type, reduced cost, and increased working time. Bumped Plasma Torch to lv 5 for more consistency in when players have access to plasma damage.

Turtlelord26 committed:
Update Melee Damage Calculations, add Damage Types

Design consistencies.

Daggers should do about 50 damage for someone with 7 Dex & Str (mods of 8), Swords 80.
Nonexotic Light weapons have a maximum total attribute modifier of 2, One-handed 3. Two-handed can go higher with Str mutipliers.
* Tools like torches get no attribute bonuses.
* Daggers receive 2x Dex but no Str, 
* Swords and Axes get 2x Str and 1x Dex
* Spears and hammers, for example, I would rate at 3x Str, no Dex

Throwing weapons set approx equivalent to their melee counterparts, dropping a level of STR mod if applicable. I gave the Axes +10 damage compared to to the knives.

Specific Notes:
The grabber knife lost 10 damage from the new baseline for its ability to start free grapples. I reworked the rest to be more streamlines.
The whip's target damage is 66.
The Fist and sledge hammer kept their obnoxiously high Str multipliers, but had base damage reduced to keep the ceiling reasonable.

Turtlelord26 committed:
Cut Sword ranges from 2 to 1m

Melee weapons, barring exotic outliers and specific reach weapons, should all have range 1.

bleehu committed:
rearranged damage types and armor to surface fall damage rule, did some style cleanup

bleehu committed:
Merge pull request #632 from bleehu/MH/play-playable/damage-type-pass

rearranged damage types and armor to surface fall damage rule, did so…

bleehu committed:
stylized and re-ordered skills document

bleehu committed:
Merge pull request #633 from bleehu/MH/play-playable/skills-pass

stylized and re-ordered skills document

Turtlelord26 committed:
Review notes: Put APL back in Thrown

+ misc and word choice.

bleehu committed:
Fixed spelling and grammar errors from review
	modified:   CharacterCreation/02_Classes.txt
	modified:   CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt
	modified:   Items/07_Items.txt

bleehu committed:
De-duplicated and re-arranged basic rules
added Primary and Secondary Stats to glossary
removed some feats rules from feats document, added feats section to basic rules
re-wrote secondary stats section
fixed b0rk'd headings not diplaying properly

More work is yet to be done; in particular, the character creation section is still duplicated. But I figured this already had enough scope creep to get bogged, so I'll continue in another PR.
	modified:   01_01_Basic_Rules.txt
	modified:   03_Glossary_of_Terms.txt
	modified:   CharacterCreation/03_Feats.txt

bleehu committed:
clarified when overcharge ammo jams

bleehu committed:
added new Duck Bob and Weave feat

bleehu committed:
Merge pull request #639 from bleehu/MH/Issue-629/overcharge-clarity

clarified when overcharge ammo jams

Eryneril committed:
At long last, Fire Extingushers

Turtlelord26 committed:
Rework Heavy Weapon Expertise 36 Pt Bonus

Again.

#572 
Now gives accuracy bonus to the "Laying Down Fire" aesthetic.

Turtlelord26 committed:
Typography

bleehu committed:
Merge pull request #640 from bleehu/MH/Issue-608/duck-bob-weave

added new Duck Bob and Weave feat

bleehu committed:
some cleaning up of weapon attachment rules in basic rules and wep attachments.
	modified:   01_01_Basic_Rules.txt
	modified:   Items/04_00_WeaponAttachments.txt

Eryneril committed:
Update 07_Items.txt

Eryneril committed:
Merge pull request #641 from bleehu/Eryneril-#560-Adding_Fire_Extinguishers

At long last, Fire Extingushers

bleehu committed:
finished doing a style pass on weapon attachments. Added flavor text to the attachments that were missing it.

Turtlelord26 committed:
Merge pull request #642 from bleehu/Turtlelord26-#572-Rework_HeavWepExp_36ptBonus

Rework Heavy Weapon Expertise 36 Pt Bonus

Turtlelord26 committed:
Moved Damage Type after attribute mods

Also gave the Power Fist and Sledgehammer minimum damage.

Turtlelord26 committed:
Review Typography

Turtlelord26 committed:
Merge pull request #630 from bleehu/Turtlelord26-Melee_Weapon_Damage_Patch

Turtlelord26 melee weapon damage patch

Turtlelord26 committed:
Word Choice and Typography

bleehu committed:
made spelling/grammar corrections requested by rb
	modified:   01_01_Basic_Rules.txt

bleehu committed:
Added some grammar clarity to weapon attachment pass
	modified:   01_01_Basic_Rules.txt
	modified:   Items/04_00_WeaponAttachments.txt

Turtlelord26 committed:
Merge branch 'dev' into Turtlelord26-#491-Explosive_Damage_Consistency

Turtlelord26 committed:
Merge pull request #575 from bleehu/Turtlelord26-#491-Explosive_Damage_Consistency

Turtlelord26 #491 explosive damage consistency

Turtlelord26 committed:
Clarify Suppression attacks wording

#646

Turtlelord26 committed:
Merge pull request #647 from bleehu/Turtlelord26-#646-Suppression_Clarity

Clarify Suppression attacks wording

bleehu committed:
Merge pull request #638 from bleehu/MH/play-playable/basic-rules-pass

De-duplicated and re-arranged basic rules

bleehu committed:
more spelling and grammar fixes on weapon attachments
	modified:   01_01_Basic_Rules.txt
	modified:   Items/04_00_WeaponAttachments.txt

bleehu committed:
Merge pull request #644 from bleehu/MH/play-playable/attachments-pass

Mh/play playable/attachments pass

bleehu committed:
clarified spotter class feat for hunters

bleehu committed:
Version bump to 0.1.6

modified Readme.md

bleehu committed:
Merge pull request #650 from bleehu/MH/Version-bump/0.1.6

Version bump to 0.1.6

bleehu committed:
Merge branch 'R_C' into dev

bleehu committed:
Merge branch 'dev' into MH/Hunter-Dog/Superbranch

bleehu committed:
grammar check for new hunter
	modified:   CharacterCreation/02_Classes.txt

Turtlelord26 commented:
Can we revert that RC to dev merge? It's erased some recent work and resurrected a few things that should stay gone.

bleehu committed:
Revert "Merge branch 'R_C' into dev"

This reverts commit bfd41f407269c989d7ed29030eb044e55de521aa, reversing
changes made to 93de60774c22a6a2c04af76309f9cb7d2898b9c1.

 Changes to be committed:
	modified:   01_01_Basic_Rules.txt
	new file:   02_01_Advanced_Movement:_Microgravity.txt
	new file:   CharacterCreation/00_New_Player_Walkthrough.txt
	new file:   CharacterCreation/01_01_Species_Human
	new file:   CharacterCreation/01_03_Species_Tundari
	modified:   CharacterCreation/02_Classes.txt
	modified:   CharacterCreation/03_Feats.txt
	modified:   CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt
	modified:   CharacterCreation/Class-Specific Documentation/Engineer Processes.txt
	deleted:    CharacterCreation/Class-Specific Documentation/Tome of Spells.txt
	modified:   Items/04_01_Melee_&_Thrown_Weapons.txt
	modified:   Items/07_Items.txt
	modified:   README.md

Turtlelord26 committed:
VAE Combat Sword Typo

Attr -> Attr mod in damage calculation. Intended change from the big damage PR, but missed in review.

Turtlelord26 committed:
Merge pull request #652 from bleehu/Turtlelord26-Melee_Weapon_Damage_Patch-2

VAE Combat Sword Typo

Turtlelord26 commented:
Formal review is ready to go, just waiting on the 561f3ef reversion to see what changes.

The only real problem I've found is the two species documents, which (again) (still) shouldn't go to RC until the rest of the Species Rework project is ready to go with them. If those two files disappear I expect to be able to give a shipit.

bleehu committed:
Merge pull request #615 from bleehu/MH/Hunter-Dog/Superbranch

Mh/hunter dog/superbranch

Turtlelord26 commented:
Strike my last

Turtlelord26 commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+humidity, wind speed and direction, and the coriollis effect making the bullet 
+swerve. The bonus to critically hit is listed below. Note that if the chance to 
+critically hit exceeds the chance to hit, it does not increase the chance to 
+hit; it only garuntees that a hit will be a critical hit. If the increase in
```
sp: guarantees

Turtlelord26 reviewed (changes requested):
RC Version 0.1.6 Technical Review

Changes by File (and at the end, project)

** items were formerly casualties of a play->RC merge, and are pretending to be reintroduced to RC here.

## Basic Rules

-Nonfunctional prose changes, reorderings, and standardizations. ✔️
-Feat description and rules moved into this document. ✔️
-Massively increased descriptive section on Attachments and other weapon customization. ✔️
-** Added Rolling the Average definition. Nonfunctional here, functional where used. ✔️

## Combat Rules

-Medium and Heavy suppressions changed to always hit armor, preparatory for Static armor changes. Technically functional. Testable. ✔️

## Damage Types

-Nonfunctional word standardization and reordering ✔️
-Added nonconductivity to regular armors. Functional. Testable. ✔️
Director's Note: This rule should also exist in the section of Combat Rules that talks about armor, or this may turn into a rules lawyer Gotcha. I would also like to know why this was introduced - what problem it's solving or value it's adding.

## **Movement: Microgravity, New Player Walkthrough

-** Added. Functional. Testable. ✔️

## Glossary

-Base Stats -> Primary stats, and added Secondary Stats. ✔️

## Species: Human, Species: Tundari

-Added files, part of Species rework. Functional. Testable. Incomplete pending resolution of the project. ❌

## **Classes

-** Added melee weapon and skill proficiencies to classes. ✔️

## Feats

-Miss Chance modifiers become Accuracy modifiers. Nonfunctional. ✔️
Director's Note: We might want these Accuracy instances capitalized.
-Added Juggler Feat. Very throwing weapons. Functional. Testable in conjunction with new Melee weapon damage. ✔️
-Added Duck, Bob, and Weave feat. Functional. Testable. Very like. ✔️
-** Added proficiency feats for melee weapon sizes and unarmed combat feats. Functional, tetable. ✔️
-Heavy weapon expertise 36 bonus changed to conditioning on consecutive attacks instead of high APL. Functional, testable. ✔️

## Skills

-Nonfunctional reorganization. ✔️

## **Doctor Procedures

-** Added Whateverectomy. ✔️

## **Tome of Spells

-** Deleted. ✔️

## Attachments

-Some rules moved to Basic Rules. Nonfunctional. ✔️
-Added nonfunctional prose, some restyling. ✔️

## Melee & Thrown

-Replaced old Hand-to-hand system with new Unarmed Strike, which grows in power via Unarmed Combat feats.
-Added Welding Torch weapon at lv 1, borrowing all stats from the plasma torch except damage type. Moved the plasma torch to lv 5. Functional, testable. ✔️
-Rebalanced melee weapon damages. Functional. Testable.
-Added explicit damage types to melee weapons. Technically functional but we've been treating them like this the whole time. Testable even. ✔️
-Modifications to the Grabber knife effect to be more in line with the new grapple. Functional, testable. Some balance concern has been noted. ✔️
-Range adjustments to the Throwing weapons, now independent of attributes. Functional. Testable. ✔️
-Individually noted throwing weapons may be used for melee attacks at regular damage but at an accuracy penalty. Functional. Testable. ✔️

## Items

-Added fire extinguisher. Functional. Testable. ✔️

## Hunter class rework: The Dog.

-Redone class flavor text, now has a fuller and more in-aesthetic theme. Nonfunctional. ✔️
-Add Hunter gains Attack Dog pet at lv 3. Functional. Testability, specifically of the level at which the Hunter gains the pet, check. Probably via metabalance project ✔️
-Add Attack dog. Refer to attack dog document for analysis.
-Remove Proficiency with Long Rifles. Functional. Testable. ✔️
-Mark Quarry ability removed. Functional. Testable. ✔️
-Completely redone class feat selection. Quite Functional
--Tracker: Targeted gunfire vulnerability debuff. Testable. ✔️
--Spotter: Increased Action economy and greater vulnerability. I don't get the crit bonus thing going on here.
--Pack Hunter I: Teamwork buffs. Testable ✔️
--Pack Hunter II: More teamwork buffs. Testable ✔️
--Bushwhacker: Anti-melee attack and perception buff. Testable ✔️
-The Dog. It's a functional change.
--Can't use non-food items. Testable. Would seem to imply that they can't use the toy, which is sad. Please address. ❓
--Gets cover via Nyoom: testable ✔️
--Can dodge melee with Reflex. Testable ✔️
--Has stats. Is missing Awareness save but this is calculable. Nonblocker. ❓
--Has skills. Testable ✔️
--Can distract with tricks. Testable ✔️ Director's query: Is this meant to be able to be used during combat? Seems off-flavor.
--Has an attack that does damage and automatically grapples. Testable. ✔️
--Has alternate damage for the Strike grapple action. Testable ✔️
-Dog Items are functional changes.
--Toy: Harder to test (no numbers), but testable. ✔️
--Harness: Testable ✔️
--Food: Testable. ✔️

dfuchs2 committed:
Added Armor Strength to Glossary

dfuchs2 committed:
Not committing directly to dev

dfuchs2 committed:
Added Armor Strength to Glossary

dfuchs2 committed:
Clarify Mendar Ability 'Colorful!'

Mendar's Colorful! Ability could be misinterpreted to allow Opaleites to turn people into followers

https://github.com/bleehu/Compound_X/issues/529

dfuchs2 committed:
Merge pull request #656 from bleehu/dfuchs2-Glossary/Armor_Strength

Added Armor Strength to Glossary

dfuchs2 committed:
Merge pull request #657 from bleehu/dfuchs2-Clarify_Mendar_Ability

Clarify Mendar Ability 'Colorful!'

dfuchs2 committed:
Add Automatic Fire Concurrent d10 rules

Added rules for concurrent d10 automatic-fire.

dfuchs2 committed:
Merge pull request #660 from bleehu/dfuchs2-Concurrent_d10_Auto

Add Automatic Fire Concurrent d10 rules

bleehu commented:
Adding a clone of this where the version bump is a separate branch
